paddle/fluid/inference/api/paddle_pass_builder.cc

```diff
@@ -399,14 +392,7 @@ void CpuPassStrategy::EnableMkldnnInt8() {
     passes_.push_back("conv_transpose_bias_mkldnn_fuse_pass");
     passes_.push_back("conv_elementwise_add_mkldnn_fuse_pass");
     passes_.push_back("conv_concat_relu_mkldnn_fuse_pass");
-    passes_.push_back("conv_relu_mkldnn_fuse_pass");
-    passes_.push_back("conv_leaky_relu_mkldnn_fuse_pass");
-    passes_.push_back("conv_relu6_mkldnn_fuse_pass");
-    passes_.push_back("conv_swish_mkldnn_fuse_pass");
-    passes_.push_back("conv_hard_swish_mkldnn_fuse_pass");
-    passes_.push_back("conv_mish_mkldnn_fuse_pass");
-    passes_.push_back("conv_hard_sigmoid_mkldnn_fuse_pass");
-    passes_.push_back("conv_gelu_mkldnn_fuse_pass");
+    passes_.push_back("conv_activation_mkldnn_fuse_pass");
```

Comment:
Could you also delete conv_relu_mkldnn_fuse_pass.cc/.h, conv_leaky_relu_mkldnn_fuse_pass.cc/.h .... there files ? Thank you !

Reply:
These passes didn't have separate files. They were all declared inside `conv_activation_mkldnn_fuse_pass` by deriving base `conv_activation` and overwriting activation name.